comRamona/assignment4/NumberOfIslandsTest.java

```diff
@@ -0,0 +1,59 @@
+package assignment4;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class NumberOfIslandsTest {
+    @Test
+    public void testNoIslands() {
+        boolean[][] grid = new boolean[][]{
+                {false, false, false},
```

Comment:
Good tests; Would suggest making constants for the false/true; which are equal length so that the maps are easily read.